src/utils/builder.ts

```diff
@@ -389,13 +389,15 @@ export function setTransitionCondition(
       (transition: Transition) => transition.id === transitionId
     );
 
-    if (foundTransition)
+    if (foundTransition) {
+      const cql = criteria.statement || criteria.builder?.cql || '';
```

Comment:
See my comment above about statement but I think the builder type should know its statement. Placing the entire CQL here is outdated and being updated in [PATHWAYS-343](https://jira.mitre.org/browse/PATHWAYS-343). I'll let you and Dan figure out who should update it but this should be a boolean valued expression (such as the statement)